bloom-housing/bloom
# fix: open house events can now be edited and work cross-browser

## Issue Overview

This PR addresses #2265 and #2263 


## Description

Fixes for both issues mentioned above: previously an existing open house event wouldn't get saved properly if you tried to edit it. In addition, open house dates (and other dates like application due date/time) would result in "Invalid Date" after adding/editing an open house in browsers like Firefox and Safari.

## How Can This Be Tested/Reviewed?

Ensure open house events can be added/edited/saved without error or data loss in Chromium-based browsers, Firefox, and Safari.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
